src/core/DataPipelineManager.ts

```
@@ -476,55 +541,87 @@ export class DataPipelineManager extends EventEmitter {
+        result = await this.applyEncryptTransformation(
+          inputData[0],
+          transformation.config
+        );
         break;
 
       default:
         // Check for custom transformation function
-        const customFunction = this.transformationFunctions.get(transformation.type);
+        const customFunction = this.transformationFunctions.get(
+          transformation.type
+        );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)